public/app/core/components/AppNotifications/AppNotificationList.test.tsx

```diff
@@ -1,194 +1,123 @@
-import { act, render, waitFor } from '@testing-library/react';
-import { Provider } from 'react-redux';
-import { MemoryRouter } from 'react-router-dom';
-import { getGrafanaContextMock } from 'test/mocks/getGrafanaContextMock';
+import { act, getWrapper, render, screen } from 'test/test-utils';
```

Comment:
We can use the `test-utils` render method which will wrap in most of the things that are required for rendering something within Grafana. In this case though we also need access to the grafanaContext, but we can override it with the `getWrapper`, and use that to enable kiosk mode